File: app/src/main/java/com/yechaoa/materialdesign/activity/FloatingActionButtonActivity.java
package com.yechaoa.materialdesign.activity;

import android.support.design.widget.FloatingActionButton;

import com.yechaoa.materialdesign.R;
import com.yechaoa.materialdesign.activity.ToolbarActivity;

import butterknife.BindView;
import butterknife.OnClick;

public class FloatingActionButtonActivity extends ToolbarActivity {

    @BindView(R.id.floating_button)
    FloatingActionButton mFloatingButton;

    @Override
    protected int getLayoutId() {
        return R.layout.activity_floating_action_button;
    }

    @Override
    protected void setToolbar() {
        mToolbar.setTitle("FloatingActionButton");
    }

    @Override
    protected void initView() {

    }

    @OnClick(R.id.floating_button)
    public void onClick() {
        finish();
    }
}

File: app/src/main/java/com/yechaoa/materialdesign/activity/FloatingActionButtonActivity.kt
package com.yechaoa.materialdesign.activity

import com.yechaoa.materialdesign.R
import com.yechaoa.materialdesign.databinding.ActivityFloatingActionButtonBinding

class FloatingActionButtonActivity : ToolbarActivity<ActivityFloatingActionButtonBinding>() {

    override fun getViewBinding(): ActivityFloatingActionButtonBinding {
        return ActivityFloatingActionButtonBinding.inflate(layoutInflater)
    }

    override fun setToolbar() {
        mToolbar.setTitle(R.string.floating_action_button)
    }

    override fun initView() {
        mBinding.floatingButton.setOnClickListener {
            finish()
        }
    }

}